.codesandbox/ci.json

```diff
@@ -36,10 +36,10 @@
   },
   "sandboxes": [
     "material-ui-issue-latest-s2dsx",
-    "github/mui/material-ui/tree/HEAD/examples/create-react-app",
-    "github/mui/material-ui/tree/HEAD/examples/create-react-app-with-typescript",
-    "github/mui/material-ui/tree/HEAD/examples/joy-cra-typescript",
-    "github/mui/material-ui/tree/HEAD/examples/base-cra-typescript"
+    "github/mui/material-ui/tree/HEAD/examples/material-cra",
+    "github/mui/material-ui/tree/HEAD/examples/material-cra-ts",
+    "github/mui/material-ui/tree/HEAD/examples/joy-cra-ts",
+    "github/mui/material-ui/tree/HEAD/examples/base-cra-ts"
```

Comment:
Not sure why CodeSandbox could not pick up the examples.

<img width="952" alt="image" src="https://user-images.githubusercontent.com/18292247/217709121-729b68d1-2eaa-46a0-b246-631ba8fc4089.png">

According to the [docs](https://codesandbox.io/docs/learn/sandboxes/ci#configuration-format), can we try

```
"sandboxes": [
  "/examples/material-cra",
  …
]
```

Reply:
Thanks! I think it's fixed now.